libs/platform/ebpf_object.c

```diff
@@ -183,64 +183,45 @@ ebpf_object_initialize(
 }
 
 void
-ebpf_object_acquire_reference(ebpf_core_object_t* object)
+ebpf_object_acquire_reference(_Inout_ ebpf_core_object_t* object)
 {
     ebpf_assert(object->base.marker == _ebpf_object_marker);
     int32_t new_ref_count = ebpf_interlocked_increment_int32(&object->base.reference_count);
     ebpf_assert(new_ref_count != 1);
 }
 
-_Requires_lock_held_(&_ebpf_object_tracking_list_lock) static void _ebpf_object_release_reference_under_lock(
-    ebpf_core_object_t* object)
+void
+ebpf_object_release_reference(_Inout_opt_ ebpf_core_object_t* object)
 {
     int32_t new_ref_count;
 
-    if (!object) {
+    if (!object)
         return;
-    }
```

Comment:
put {} back in per coding style

Reply:
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes

Guidelines say no {} for single line if statements.

Reply:
It says:
> Single-line if/else/loop blocks MUST be enclosed in braces.